stratum/hal/lib/tdi/dpdk/dpdk_sde_target.cc

```diff
@@ -132,7 +132,8 @@ ::util::Status TdiSdeWrapper::AddPort(int device, int port, uint64 speed_bps,
   port_attrs->port_dir = PM_PORT_DIR_DEFAULT;
   port_attrs->port_in_id = port;
   port_attrs->port_out_id = port;
-  port_attrs->net_port = config.packet_dir;
+  port_attrs->net_port =
+      config.packet_dir == hal::PortConfigParams::DIRECTION_NETWORK ? 1 : 0;
```

Comment:
If we do a conditional instead of a direct assignment here, we don't really need to ensure that `DIRECTION_NETWORK` has a specific value.